src/dev/ci_setup/setup.sh

```diff
@@ -149,6 +149,11 @@ else
   echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
 fi
 
+###
+### skip chomium download, use the system chrome install
+###
+export PUPPETEER_SKIP_CHROMIUM_DOWNLOAD true
```

Comment:
dont need the equals sign?
as in `export PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true`

Reply:
Yeah, definitely does, oops